src/App.vue

```diff
@@ -5,43 +5,27 @@
     <div class="wrap-outer-content layout-band">
       <div class="wrap-content">
         <div class="layout-band">
-          <SearchForm @searched="doSearch" />
+          <SearchForm @searched="receiveSearch" />
           <SearchMetadata
             v-if="showSearchMetadata"
             v-model:query.lazy="query"
             v-model:status="status"
             v-model:hits="hits"
           />
+          <div v-if="status.loading" class="panel panel-info">
+            <div class="panel-heading">Loading...</div>
+          </div>
+          <div v-if="status.errored" class="panel panel-danger">
+            <div class="panel-heading">Something went wrong</div>
+            <div class="panel-body">
+              <p>{{ status.error_message.message }}</p>
+              <pre>{{ status.error_message }}</pre>
+            </div>
+          </div>
         </div>
         <div v-bind:class="layoutBand">
-          <div v-if="sidebarLeft" class="col1q">
-            <Facet
-              v-for="facet in facets"
-              v-bind:facet="facet"
-              v-bind:key="facet"
-            />
-          </div>
           <div v-bind:class="layoutContent">
-            <div v-if="status.loading" class="panel panel-info">
-              <div class="panel-heading">Loading...</div>
-            </div>
-            <div v-if="status.errored" class="panel panel-danger">
-              <div class="panel-heading">Something went wrong</div>
-              <div class="panel-body">
-                <p>{{ status.error_message.message }}</p>
-                <pre>{{ status.error_message }}</pre>
-              </div>
-            </div>
-
-            <router-view />
-
-            <Item
-              v-for="(result, index) in results"
-              v-bind:result="result"
-              v-bind:index="index"
-              v-bind:key="result.id"
-            />
-            <Pagination v-if="showPagination" msg="Pagination bar" />
+            <router-view @results="receiveSummary" />
```

Comment:
Is this the same `<router-view>` tag used by all routes?

Reply:
Yeah, this is the one slot in the template where any route would display (so the results and record views currently)